extensions/default/src/getCustomizationModule.tsx

```diff
@@ -29,315 +30,267 @@ export default function getCustomizationModule({ servicesManager, extensionManag
   return [
     {
       name: 'helloPage',
-      merge: 'Append',
       value: {
-        id: 'customRoutes',
-        routes: [
-          {
-            path: '/custom',
-            children: () => <h1 style={{ color: 'white' }}>Hello Custom Route</h1>,
+        customRoutes: {
+          routes: {
+            $push: [
+              {
+                path: '/custom',
+                children: () => <h1 style={{ color: 'white' }}>Hello Custom Route</h1>,
+              },
+            ],
           },
-        ],
+        },
       },
     },
-
     // Example customization to list a set of datasources
     {
       name: 'datasources',
-      merge: 'Append',
       value: {
-        id: 'customRoutes',
-        routes: [
-          {
-            path: '/datasources',
-            children: DataSourceSelector,
+        customRoutes: {
```

Comment:
Could we get better naming on this - even if we don't do it now, at least it should be documented that in the next major release/version that this will be being renamed to the standardized naming.

Reply:
As discussed "Routes.CustomRoutes"